src/TwigComponent/src/Command/TwigComponentDebugCommand.php

```diff
@@ -28,7 +27,6 @@
 use Symfony\UX\TwigComponent\Twig\PropsNode;
 use Twig\Environment;
 
-#[AsCommand(name: 'debug:twig-component', description: 'Display components and them usages for an application')]
```

Comment:
Are most of the commands in the symfony repository using this attribute ?

Reply:
They are, yes. I opted to remove to reduce duplication.